### DISCUSSION TITLE
The Kamon APM UI doesn't work

### DISCUSSION THREAD

**Discussion by mkemaldurmus** [kamon-io/Kamon#1279] [General] [answered]
I have been working on APM installation, but I encountered an issue the  UI not refreshing. I can't see anything.
<img width="1728" alt="Screenshot 2023-05-23 at 12 44 42" src="https://github.com/kamon-io/Kamon/assets/45178489/f3dcb0f2-820c-408e-bf22-e0f20f24ff8b">

**Comment by ivantopo** [comment]
Looking into it right away!

**Reply by ivantopo** [reply, marked as answer]
Hey @mkemaldurmus, give it a try again. There was a clogged service on the read side. Things are good now!

**Reply by mkemaldurmus** [reply]
Thanks it worked

**Reply by mkemaldurmus** [reply]
@ivantopo same issue again

**Reply by ant8e** [reply]
+1

**Reply by ivantopo** [reply]
Hey folks, I think now the issue is fixed for good. Long story short is that some I had to do a full restart of our Clickhouse cluster for this to really get back on track :cry:
